src/reporters/base-reporter.js

```diff
@@ -170,6 +175,27 @@ export default class BaseReporter {
     }
   }
 
+  getSizeUnit(iterations: number) {
```

Comment:
Why not use something already available like https://yarnpkg.com/en/package/humanize-bytes instead of rolling your own?

Reply:
Oh, yes. Probably a good idea. Retrospect tells me I was trying to be too clever here.